demos/ros_examples/scripts/joint_space_velocity_control.cpp

```diff
@@ -0,0 +1,100 @@
+#include <chrono>
+#include <ros/ros.h>
+#include <std_msgs/Float64MultiArray.h>
+#include <sensor_msgs/JointState.h>
+#include <state_representation/space/cartesian/CartesianPose.hpp>
+#include <state_representation/space/cartesian/CartesianTwist.hpp>
+#include <state_representation/robot/JointVelocities.hpp>
+#include <dynamical_systems/Linear.hpp>
+#include <robot_model/Model.hpp>
+
+using namespace state_representation;
+using namespace dynamical_systems;
+using namespace robot_model;
+using namespace std::chrono_literals;
+
+namespace {
+class RobotInterface {
+private:
+  Model robot_;
+  ros::Subscriber subscriber_;
+  ros::Publisher publisher_;
+
+public:
+  JointState joint_state;
```

Comment:
Do you need that to be public? It does not seemed to be used anywhere afterwards